crates/red_knot_python_semantic/resources/mdtest/literal/literal.md

```diff
@@ -75,6 +75,8 @@ Literal: _SpecialForm
 ```py
 from other import Literal
 
+# TODO: we should not emit this error once we understand Literal[..] annotations
+# error: [annotation-with-invalid-expression]
```

Comment:
Hmm, I think this is a regression? We added support for `Literal[...]` annotations in https://github.com/astral-sh/ruff/pull/13874

Reply:
Note that this is not `typing.Literal`, but `other.Literal`.

Reply:
Oh. In that case, I think this is a true positive, and we should remove the TODO?

Reply:
Yes, seems like I was similarly confused when I added the TODO. Thanks!